packages/block-library/src/gallery/edit.js

```diff
@@ -696,7 +694,4 @@ function GalleryEdit( props ) {
 		</>
 	);
 }
-export default compose( [
-	withNotices,
-	withViewportMatch( { isNarrow: '< small' } ),
-] )( GalleryEdit );
+export default withViewportMatch( { isNarrow: '< small' } )( GalleryEdit );
```

Comment:
@geriux, the ` withViewportMatch` value is only used in [the `gallery.native.js` file](https://github.com/WordPress/gutenberg/blob/b11bdc0645a0503437389c29d39ac266ba6cd2bf/packages/block-library/src/gallery/gallery.native.js#L42). Would it makes sense to move this HoC there?

Reply:
I think it makes sense yes, we could move it. Do you want me to help with that once this PR lands in trunk? Or should we address it in this PR?

Reply:
Let’s do it in one go. I’ll push updates here shortly.

Thanks, @geriux!

Reply:
Sounds good! Thank you!

Reply:
Updated in [0fc0e77](https://github.com/WordPress/gutenberg/pull/64384/commits/0fc0e77765c8e6a976cb3214c569f6c259a0c53a).